backend/core/models.py

```
@@ -4,7 +4,7 @@
 import hashlib
 from datetime import date, datetime
 from pathlib import Path
-from typing import Self, Type, Union
+from typing import Self, Type, Union, List
```

Ruff (0.11.9)

`typing.Type` imported but unused

Remove unused import: `typing.Type`

(F401)